src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -116,6 +123,15 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.custom_command('list', 'list_network_rules')
         g.custom_command('remove', 'remove_network_rule')
 
+    with self.command_group('storage account blob-service-properties', blob_service_mgmt_sdk,
+                            custom_command_type=storage_account_custom_type,
+                            resource_type=ResourceType.MGMT_STORAGE, min_api='2019-04-01') as g:
```

Comment:
Blob Service Properties are supported for a long time,  and the correct min api version constraint for blob service properties should be 2018-07-01 I think.
But for changefeed feature it should be 2019-04-01.

Reply:
updated.

---
In reply to: [354647458](https://github.com/Azure/azure-cli/pull/11488#discussion_r354647458) [](ancestors = 354647458)